modules/installation-azure-permissions.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * installing/installing_azure/installing-azure-account.adoc
 // * installing/installing_azure/installing-azure-user-infra.adoc
+// * installing/installing_azure_stack_hub/installing-azure-stack-hub-user-infra.adoc
 
 [id="installation-azure-permissions_{context}"]
 = Required Azure roles
```

Comment:
The doc says ASH requires `User Access Administrator`

I don't think this is accurate for ASH. According to [this doc](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-manage-permissions?view=azs-2102) there are three kinds of roles:

```
Owner: can manage everything, including access to resources.
Contributor: can manage everything, except access to resources.
Reader: can view everything, but can't make any changes.
```

I would think the `Owner` role would be required for creating the service principal, which has a contributor role, and then that contributor role can be used for everything else.

Reply:
I created a new module for required ASH roles, and updated the content to require the `Owner` role.